js/login.js

```diff
@@ -0,0 +1,111 @@
+import { validateEmail, isPasswordValid, isEmpty } from "/js/common/formValidator.js";
+
+const emailInput = document.getElementById("inputEmail");
+const pwdInput = document.getElementById("inputPwd");
+
+const emailValidError = document.getElementById("emailValidError");
+const pwdValidError = document.getElementById("pwdValidError");
+
+const loginBtn = document.getElementById("loginBtn");
+
+const EMAIL_EMPTY_ERROR_MSG = "이메일을 입력해주세요.";
+const EMAIL_VALID_ERROR_MSG = "잘못된 이메일 형식입니다.";
+const PWD_EMPTY_ERROR_MSG = "비밀번호를 입력해주세요.";
+const PWD_LENGTH_ERROR_MSG = "비밀번호를 8자 이상 입력해주세요.";
+
+// 에러 메세지 표시
+function showError(inputEl, errorEl, message) {
+  inputEl.classList.add('error');
+  inputEl.classList.remove("success");
+  errorEl.textContent = message;
+};
+
+// 에러 메세지 제거
+function clearError(inputEl, errorEl) {
+  inputEl.classList.remove('error');
+  inputEl.classList.add('success');
+  errorEl.textContent = '';
+};
+
+/**
+ * 입력값들의 유효성을 검사한 후,
+ * 모든 항목이 유효할 경우 회원가입 버튼이 활성화
+ * 그렇지 않으면 비활성화
+ */
+function toggleButtonState() {
+  const isEmailValid = !isEmpty(emailInput.value) && validateEmail(emailInput.value);
+  const isPwdValid = !isEmpty(pwdInput.value) && isPasswordValid(pwdInput.value);
+
+  const isFormValid = isEmailValid && isPwdValid;
+
+  if (isFormValid) {
+    loginBtn.setAttribute("aria-disabled", "false");
+    loginBtn.classList.remove("disabled");
+  } else {
+    loginBtn.setAttribute("aria-disabled", "true");
+    loginBtn.classList.add("disabled");
+  }
+}
```

Comment:
💊 제안
저는 여러번 사용되는 값이거나 변수에 할당했을 시 가독성에 이점이 있는 경우 변수에 할당하는 것을 추천드리는데요.
지금의 경우는 위에 해당하는지 잘 모르겠습니다.
이러한 경우 함수로 빼시는 것도 가독성 측면에서 좋습니다!

```suggestion
function checkFormValid() {
  const isEmailValid = !isEmpty(emailInput.value) && validateEmail(emailInput.value);
  const isPwdValid = !isEmpty(pwdInput.value) && isPasswordValid(pwdInput.value);
  
  return isEmailValid && isPwdValid;
}

function toggleButtonState() {
  if (checkFormValid()) {
    loginBtn.setAttribute("aria-disabled", "false");
    loginBtn.classList.remove("disabled");
  } else {
    loginBtn.setAttribute("aria-disabled", "true");
    loginBtn.classList.add("disabled");
  }
}
```

추가적으로 해당 로직은 아래처럼도 작성 가능합니다.

```js
function toggleButtonState() {
    const isFormInvalid = !checkFormValid();

    loginBtn.setAttribute("aria-disabled", `${isFormInvalid}`);
    loginBtn.classList.toggle("disabled", isFormInvalid);
}
```